healthScore/healthConfig/settings.py

```diff
@@ -82,7 +82,7 @@
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'HealthScore',
         'USER': 'root',
-        'PASSWORD': '', # change it with your sql server password for local testing
+        'PASSWORD': 'Wyx@0809ml', # change it with your sql server password for local testing
```

Comment:
Please undo this change

Reply:
is it ok that you disapprove of this change when merging?

Reply:
Okay no issues, I will fix it